### PR TITLE
Implement todo list

### PR DESCRIPTION
**Pull request by benshidanny11**
> In this milestone I:
- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Created an index.html file.
- Created an index.js file and set an array of some simple to-do tasks (array of objects).
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- Implemented feature for listing all tasks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,29 @@
 {
-  "name": "todolist",
+  "name": "todo-list",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "private": "true",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "webpack"
```
Comment (DeliceLydia) [inline package.json]: - [ ] I suggest that you add the script to start the server this one you have should be `build` that runs webpack you can use
`"start": "webpack serve --open"`,
 `"build": "webpack"`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @benshidanny11, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- No linter errors
- Awesome README


## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by zainsadaqat — APPROVED** [review]
# Hello @benshidanny11  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (benshidanny11) [reply package.json]: Hello @DeliceLydia, Thanks for giving me feedback. I am going to implement it.

**Comment by benshidanny11** [comment]
Hello @zainsadaqat, Thanks for approving my project 🙏

**Pull request merged**